### PR TITLE
Bug fix?

### PR DESCRIPTION
**Pull request by CooperMoore9**
What? Really? is that fixed? its not breaking anymore?

### CODE REVIEW

File: dist/allProjectsObject.js [hunk 1]
```diff
@@ -1,10 +1,7 @@
 "use strict";
-//Create new Project object when new project is added.
-//Probable gonna need an array of projects that can be selected as well.
-//Project object needs title, and array of task objects that are stored within the project.
-//When tasks are added, create task Object, and store in current Project Object
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.Task = exports.Project = exports.AllProjects = void 0;
+const uuid_1 = require("uuid");
```
Comment (Crowe7) [inline dist/allProjectsObject.js:4]: dont use the require syntax if you can help it. use `import` at the top. also what is this actually doing?

File: dist/uiUpdater.js [hunk 2]
```diff
@@ -88,6 +88,7 @@ function deleteProject(project) {
     if (_1.projectsSetup.projects[0]) {
         project.name = _1.projectsSetup.projects[0].name;
         taskHeader.textContent = `${project.name} Tasks`;
+        refreshTasks();
```
Comment (Crowe7) [inline dist/uiUpdater.js:91]: why does this work?

**Review by Crowe7 — CHANGES_REQUESTED** [review]

File: src/uiUpdater.ts [hunk 3]
```diff
@@ -106,11 +108,13 @@ export function loopTasks(selectedProject: Project) {
         if(projectsSetup.projects[0]){
             project.name = projectsSetup.projects[0].name;
             taskHeader.textContent = `${project.name} Tasks`;
+            refreshTasks()
```
Comment (Crowe7) [inline src/uiUpdater.ts:111]: why does this work?

**Review by Crowe7 — APPROVED** [review]

File: src/uiUpdater.ts (hunk 3 above)
Comment (CooperMoore9) [reply src/uiUpdater.ts]: Redundant and removed + ratio

**Pull request merged**